runtime/onert/core/include/ir/Index.h

```diff
@@ -75,6 +78,10 @@ inline std::ostream &operator<<(std::ostream &o, const ModelIndex &i)
   return _index_print_impl(o, "MODEL", i);
 }
 
+inline std::ostream &operator<<(std::ostream &o, const OriginIndex &i)
+{
+  return _index_print_impl(o, "", i);
```

Comment:
Did you intend to have no prefix?

Reply:
Yes. Origin is not printed during iterating operators, it is printed during iterating operands. No prefix looks better. I will push another PR for dumping graph. We can enhance log printing if necessary in forecoming PR.